content/courses/program-security/type-cosplay.md

```diff
@@ -97,29 +100,30 @@ pub mod type_cosplay_insecure {
 
 #[derive(Accounts)]
 pub struct AdminInstruction<'info> {
+    /// CHECK: This account is not being validated by Anchor
     admin_config: UncheckedAccount<'info>,
     admin: Signer<'info>,
 }
 
-#[derive(BorshSerialize, BorshDeserialize)]
+#[derive(AnchorSerialize, AnchorDeserialize, InitSpace)]
 pub struct AdminConfig {
     admin: Pubkey,
 }
 
-#[derive(BorshSerialize, BorshDeserialize)]
+#[derive(AnchorSerialize, AnchorDeserialize, InitSpace)]
 pub struct UserConfig {
     user: Pubkey,
 }
 ```
 
-#### Add account discriminator
+#### Add Account Discriminator
 
-To solve this, you can add a discriminant field for each account type and set
-the discriminant when initializing an account.
+To resolve this, add a discriminant field for each account type and set the
+discriminant when initializing an account.
```

Comment:
More of complaint with the old lesson, but is a `discriminant` the same thing as a discriminator? This is confusing.

Reply:
One of the aims of our docs is to avoid confusion - we should call it out explicitly. Add an `info` ContentBox with:

> While they sound similar, a Rust **discriminant** isn't the same thing as an Anchor **discriminator**! Rust itself has the concept of discriminants which are used to manage enums.

Or something similar - you're an excellent writer and can use whatever words you feel best illuminates the difference.